### PR TITLE
checker: allow to ignore some checking items

### PR DESCRIPTION
**Pull request by GregoryIan**
### What problem does this PR solve? 
allow to ignore some checking items

### What is changed and how it works?
add a `IgnoreCheckingItems` configuration, and use it to filter checking items in `checker`

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)
    - set up a dm cluster
    - start a config by task
    - check output in log

Code changes
 - Has exported variable/fields change

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@july2993 PTAL

File: checker/cmd.go [hunk 1]
```diff
@@ -27,7 +27,16 @@ var (
 
 // CheckSyncConfig checks synchronization configuration
 func CheckSyncConfig(ctx context.Context, cfgs []*config.SubTaskConfig) error {
-	c := NewChecker(cfgs)
+	if len(cfgs) == 0 {
+		return nil
+	}
+
+	checkingItems := config.FilterCheckingItems(cfgs[0].IgnoreCheckingItems)
```
Comment (july2993) [inline checker/cmd.go]: unclear why just get from index 0

**Comment by GregoryIan** [comment]
@july2993 @csuzhangxc PTAL again

**Pull request merged**